pystiche_papers/ulyanov_et_al_2016/_modules.py

```diff
@@ -150,6 +204,20 @@ def conv_block(
 
 
 class JoinBlock(nn.Module):
+    r"""JoinBlock from :cite:`ULVL2016`.
```

Comment:
Just to be sure: could you check the paper and verify that the join blocks in the paper comprise the norm layers?

Reply:
The join blocks contain the norm layers, but also an upsample block. I will indicate this in the documentation that this upsample block is not included.